src/main/java/racingcar/controller/ConsoleController.java

```diff
@@ -0,0 +1,62 @@
+package racingcar.controller;
+
+import java.util.List;
+import racingcar.dto.GameDto;
+import racingcar.dto.GameResultDto;
+import racingcar.dto.NamesDto;
+import racingcar.dto.ResultDto;
+import racingcar.dto.TryCountDto;
+import racingcar.service.GameService;
+import racingcar.view.RacingCarView;
+
+public class ConsoleController {
+    private final RacingCarView racingCarView;
+    private final GameService gameService;
+
+    public ConsoleController(RacingCarView racingCarView, GameService gameService) {
+        this.racingCarView = racingCarView;
+        this.gameService = gameService;
+    }
+
+    public void run() {
+        final List<String> names = receiveCarNames().getNames();
+        final int tryCount = getTryCount();
+        final GameDto gameDto = new GameDto(names, tryCount);
+        final GameResultDto gameResult = gameService.play(gameDto);
+        final List<ResultDto> racingCars = gameResult.getRacingCars();
+        printCarStatuses(racingCars);
+        final String winners = gameResult.getWinners();
+        printWinners(winners);
```

Comment:
사소하지만 조금 더 직관적으로 보일 수 있다고 생각합니다!

```suggestion
        printResult(gameResult);
        
        private void printResult(GameResult gameResult) {
            racingCarView.printRacingProgress(gameResult.getRacingCars());
            racingCarView.printWinners(gameResult.getWinners());
        }
```